packages/backend/init-action/helpers/aws.js

```diff
@@ -0,0 +1,31 @@
+const S3 = require('aws-sdk/clients/s3');
+
+// TODO: figure out where this should go
+const BUCKET = 'gitchaos';
+
+
+const uploadHistory = async ({
+  accessKeyId,
+  secretAccessKey,
+  body,
+  sha: Key,
+}) => {
+  const s3 = new S3({
+    accessKeyId,
+    secretAccessKey,
+  });
+
+  const upload = s3.upload({
+    Bucket: BUCKET,
+    Body: JSON.stringify(body),
+    Key,
+  }).promise();
+
+  const { Location: location } = await upload;
+
+  return location;
```

Comment:
Question here, 

`const { Location: location } = await upload;`

Means it creates a `const Location` with the value `location`?

Because we're returning `location` but I think it should be uppercase `Location`. Unsure though.

Reply:
Other way around I think - https://wesbos.com/destructuring-renaming/

Reply:
I didn't want to return it upercase as that's bleurgh - but I also can't do the destructure in the return (can't return with an assignment)

Open to ways to make it less gnarly though